src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -456,8 +456,9 @@ void ClusterResourceScheduler::AddLocalResourceInstances(
 
   for (size_t i = 0; i < instances.size(); i++) {
     node_instances->available[i] += instances[i];
-    node_instances->total[i] =
-        std::max(node_instances->total[i], node_instances->available[i]);
+    node_instances->total[i] += instances[i];
```

Comment:
@wuisawesome Hey Alex do you know what was the reasoning for this logic?

It looks like this is fragile when we add local resource instances multiple times imagine this; 

```
total: 1000 avail: 1000
one of task uses 1 resource
total: 1000 avail: 999
and then we add new 1000 local resources
total: 1999 avail: 1999 -> this is wrong
```

This causes the issue from the placement group. Do you know why we need to make logic like this? What was the original reasoning, and is there any side effect of my change?